lib/govuk_tech_docs/api_reference/templates/schema.html.erb

```diff
@@ -11,7 +11,7 @@
 <td><%= property[0] %></td>
 <td><%= property[1].type %></td>
 <td><%= property[1].required.present? %></td>
-<td><%= markdown(property[1].description) %></td>
+<td><%= property[1].description_html %></td>
```

Comment:
While you're here, could you apply the same changes to this file as you did in the "Rename some variables for clarity" commit?

Reply:
I don't think it's necessary to do a change here as it's done in Tijmen's commit which is next.